cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/DiagnosticsProcessor.java

```diff
@@ -40,51 +40,60 @@ public DiagnosticsProcessor(final Configuration configuration) {
             throw new IllegalStateException("Configuration does not have any module defined.");
         }
 
-        initReporters(configuration.reporters);
-        initModules(configuration.modules);
+        if (configuration.global == null) {
+            throw new IllegalStateException("Configuration does not have global configuration defined.");
```

Comment:
Just a side note - this is going to break compatibility with previous/existing configuration files, right?